src/metrics/index.js

```diff
@@ -35,13 +36,17 @@ import {
   MySegmentsCollector,
   ClientCollector
 } from './Collectors';
+import { OPTIMIZED, STANDALONE_MODE } from '../utils/constants';
 
 const log = logFactory('splitio-metrics');
+const IMPRESSIONS_REFRESH_COUNT = 300000; // 5 minutes
```

Comment:
IMPRESSIONS_COUNT_RATE ? The refresh word was very crappily used in the past here.